lib/webauthn/credential_request_options.rb

```diff
@@ -17,7 +17,11 @@ def initialize(allow_credentials: [], extensions: nil, user_verification: nil)
     end
 
     def to_h
-      options = { challenge: challenge, allowCredentials: allow_credentials }
+      options = {
+        challenge: challenge,
+        client_timeout: client_timeout,
```

Comment:
Despite what we call in the gem configuration, the hash key still needs to be `timeout`, because that's what the browser expects (https://www.w3.org/TR/webauthn/#dom-publickeycredentialcreationoptions-timeout).

Reply:
agree on this